test/unit/notifier_test.exs

```diff
@@ -14,12 +17,14 @@ defmodule NotifierTest do
     @impl BoomNotifier.Notifier
     def notify(error_info, options) do
       subject_prefix = Keyword.get(options, :subject)
-      to_respond_pid = Keyword.get(options, :sender_pid)
+      to_respond_pid = Keyword.get(options, :sender_pid_name) |> Process.whereis()
```

Comment:
I get the rule is to use the pipe operator for three or more chained calls, but in my opinion this line is looks better as it because of the line avobe